packages/esm-patient-banner-app/src/config-schema.ts

```diff
@@ -45,6 +45,45 @@ export const configSchema = {
         'Specifies the paper size for printing the sticker. You can define the size using units (e.g., mm, in) or named sizes (e.g., "148mm 210mm", "A1", "A2", "A4", "A5").',
       _default: 'A4',
     },
+    printMultipleStickers: {
+      enabled: {
+        _type: Type.Boolean,
+        _description: 'Whether to allow printing multiple patient ID stickers',
+      },
+      totalStickers: {
+        _type: Type.Number,
+        _description: 'The number of patient ID stickers to print',
+      },
+      stickerColumnsPerPage: {
```

Comment:
I agree with you @ibacher.  I also wanted to implement it that way.  
The main reason why i chose to use this alternative is to give a user control of where they want to insert a page-break so that sticker content doesn't overflow to the next page.  For example if a user specifies 2 rows per page, the page break with be inserted after the second row. 

Below are the reasons why i resorted to this.
- page size has a variety of possibilities - known page sizes like A0, A1, A4, A5, A6, legal, etc needed to be mapped to corresponding length and width.  Also page sizes might vary in different printers.
- The sticker size is dynamic depending on the number of patient fields to display. It might become shorter or longer.  
- Diversity of printers to be used VS different sizes used in different implementations.

Creating a function that could cater for all the use-cases was seemingly complex compared to merely allow a user to explore what works for him/her.

Reply:
I am happy to fix this in a future PR

Reply:
> in would be roughly one label per "page", with each page size being the size of the sticker.

@ibacher this serves one use-case.  If its a thermal printer, then yes we are printing one label for page but with regular printers, we shall be printing multiple labels per page
You an also follow this [slack thread](https://openmrs.slack.com/archives/CHP5QAE5R/p1729776817501349) about the requirement.

Reply:
I have created a [ticket](https://openmrs.atlassian.net/browse/O3-4197) to follow this up